flamedisx/xenon/x1t_sr1.py

```diff
@@ -21,7 +21,7 @@
 DEFAULT_P_DPE = 0.219
 DEFAULT_EXTRACTION_EFFICIENCY = 0.96
 
-DEFAULT_ELECTRON_LIFETIME = 641e3
+DEFAULT_ELECTRON_LIFETIME = 11600000
```

Comment:
Is it a good idea to have such a big default number for elife? But I guess @eangelino sets this via the config file anyway.
```DEFAULT_ELECTRON_LIFETIME = 11600000```

Reply:
this is the default value for nT. I modified it in the file because, even if in the config the nT value, it kept resetting the default to the 1t one.. not really understood why.. :thinking:

Reply:
Ok! modified with commit [e85cf9a](https://github.com/FlamTeam/flamedisx/pull/288/commits/e85cf9abdb5ec0a84075be25d8ae32e33e0b147b) 
I am going to resolve this conversation 
Thanks a lot Pueh! :smiling_face_with_three_hearts: